final-project/scripts/blog.js

```diff
@@ -0,0 +1,139 @@
+import Api from './api';
+
+class Blog {
+    constructor() {
+        this.blogApi         = new Api('https://wt-4662f45b9eefda7172b747b28d23efdb-0.sandbox.auth0-extend.com/blog');
+        this.articles        = [];
+        this.slider          = document.querySelector('#blog__slider');
```

Comment:
I would have probably used a class like `.js-blog__slider` instead of an Id, just in case you wanted to use this "behaviour" in other places